datafusion/physical-expr/src/expressions/cast.rs

```diff
@@ -33,7 +32,12 @@ use datafusion_common::{DataFusionError, Result};
 use datafusion_expr::ColumnarValue;
 
 /// provide DataFusion default cast options
-pub const DEFAULT_DATAFUSION_CAST_OPTIONS: CastOptions = CastOptions { safe: false };
```

Comment:
This change is necessary because FormatOptions isn't const constructible, it is somewhat an accident that CastOptions has historically been - any non-trivial additional options would have broken this